### ISSUE TITLE
comment on select models check box

### ISSUE THREAD

**Issue by elray1** [reichlab/predtimechart#37]
The checkbox after the “select models” menu item is designed as a “select all” box, but it’s not really clear that that is what it is, or how that interacts with the current selected models.

**Comment by matthewcornell** [comment]
I agree that it's unintuitive. Maybe the checkbox should be a button, like "Shuffle Colors"?
